adagrams/game.py

```diff
@@ -1,11 +1,152 @@
+import random
+LETTER_QUANTITY = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}  
+LETTER_POINT_SYSTEM = {
+    'A': 1, 
+    'B': 3, 
+    'C': 3, 
+    'D': 2, 
+    'E': 1, 
+    'F': 4, 
+    'G': 2, 
+    'H': 4, 
+    'I': 1, 
+    'J': 8, 
+    'K': 5, 
+    'L': 1, 
+    'M': 3, 
+    'N': 1, 
+    'O': 1, 
+    'P': 3, 
+    'Q': 10, 
+    'R': 1, 
+    'S': 1, 
+    'T': 1, 
+    'U': 1, 
+    'V': 4, 
+    'W': 4, 
+    'X': 8, 
+    'Y': 4, 
+    'Z': 10
+    }
+
 def draw_letters():
-    pass
+    
+    #iterates over LETTER_QUANTITY dict to make a list of all letters with exact occurrences
+    master_letter_bank = []
+    for letter, frequency in LETTER_QUANTITY.items():
+        for i in range(frequency):
+            master_letter_bank.append(letter)
+
+    # uses a randomizing operator on above list and removes that letter from the list so it cannot be reselected
+    drawn_hand = []
+    while len(drawn_hand) < 10:
+        new_letter = random.choice(master_letter_bank)
+        drawn_hand.append(new_letter)
+        master_letter_bank.remove(new_letter)
+
+    #result is a list with ten selected letters for user's hand    
+    return drawn_hand
+
+
+def uses_available_letters(word, drawn_hand):
+
+    #makes a dictionary called max_frequency from drawn_hand that has letter and frequency
+    valid_word = False
```

Comment:
This variable is not needed and the way it's being used right now contains a bug. Currently, you are changing this variable to `True` as soon as any character in the word is valid rather than after looking at all of the characters in the word. You happen to return `False` immediately as soon as you know the word is not valid so by the time you do `return valid_word`, it should be `True` anyway, but it gets set to `True` way earlier than it's supposed to. Instead, you can get rid of all of the places where you use the variable and change your last return statement to `return True` and your code would still work.